keps/sig-node/2400-node-swap/README.md

```diff
@@ -647,13 +681,17 @@ high level (needs more precise definitions) those may be things like:
   - 99,9% of /health requests per day finish with 200 code
 -->
 
+N/A
+
 ###### Are there any missing metrics that would be useful to have to improve observability of this feature?
 
 <!--
 Describe the metrics themselves and the reasons why they weren't added (e.g., cost,
 implementation difficulties, etc.).
 -->
 
+N/A
```

Comment:
This seems like a "yes" since the beta graduation criteria have a stub for "add metrics", right?

Reply:
I don't think we will be adding any metrics to Kubernetes to track performance of this; rather, we will identify preexisting metrics that can be used.